docs/content/wiring.md

```diff
@@ -153,6 +153,32 @@ For example, given the install step above, we can use the `MYSQL_URL` with the h
 
 Just like in the case of credentials and parameters, the value of the `bundle.outputs.MYSQL_URL` reference will be rewritten in the YAML before the helm mixin is invoked.
 
+## Wiring Images
+
+In the `porter.yaml`, you can define what images will be used within the bundle with the `images` section:
+
+```yaml
+images:
+  ALIAS:
+    description: A very useful image
+    imageType: docker # porter.yaml can default this to docker if we aren't already
```

Comment:
Should the comment change to say porter is defaulting it to docker (if that's the case)?